tutorials/python/posts/basic-syntax.md

```diff
@@ -1 +1,116 @@
-Lorem ipsum dolor sit amet, consectetur adipisicing elit. At quisquam, doloremque laboriosam, architecto iure accusamus ut molestiae eos cum vel suscipit accusantium quidem natus, excepturi commodi. Esse quo quibusdam dolorem.
\ No newline at end of file
+#Basic Python Syntax
```

Comment:
Have a space between # and word.

Refer to [this](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for the correct Markdown syntax

Reply:
> @Chiouonthis Thanks! Unfortunately, we don't have content style guidelines just yet, but you can take a look at our other tutorials, just try to follow a similar style guideline to that.
> 
> Sorry for the inconvenience

Yeah, so I did use W3Schools as a reference: https://www.w3schools.com/python/python_syntax.asp

**Please don't ask someone to refer to guidelines that do not exist.** I'm happy to write up some of the additions you proposed, but I do not think it is right for you to say that I did not meet some standard that isn't even established.